src/prpl.ts

```diff
@@ -71,17 +76,30 @@ const isServiceWorker = /service-worker.js$/;
  * Return a new HTTP handler to serve a PRPL-style application.
  */
 export function makeHandler(root?: string, config?: Config): (
-    request: http.IncomingMessage, response: http.ServerResponse) => void {
+    request: http.IncomingMessage,
+    response: http.ServerResponse,
+    next?: express.NextFunction) => void {
   const absRoot = path.resolve(root || '.');
   console.info(`Serving files from "${absRoot}".`);
+
   const builds = loadBuilds(absRoot, config);
+
   const cacheControl = (config && config.cacheControl) || 'max-age=60';
   const unregisterMissingServiceWorkers =
       (config && config.unregisterMissingServiceWorkers != undefined) ?
       config.unregisterMissingServiceWorkers :
       true;
+  const forwardErrors = config && config.forwardErrors;
+
+  return function prplHandler(request, response, next) {
```

Comment:
Could just be an anonymous function.

Reply:
Naming it means a stack trace will be more readable (https://github.com/Polymer/prpl-server-node/pull/8#discussion_r116349031).

Reply:
TIL.